### ISSUE TITLE
pcloud another IP address on hosting

### ISSUE THREAD

**Issue by gbyuk06** [vb6rocod/utils#59]
Hello Friend, how to fix pcloud "This link was generated for another IP address. Try previous step again." on hosting , working on localhost ?? thanksyouu

**Comment by vb6rocod** [comment]
localhost work for me.

**Comment by gbyuk06** [comment]
yes it works on localhost, but on the website/hosting it doesn't work

**Comment by racazahassem09191990** [comment]
Please update admin
